README.md

```diff
@@ -1,32 +1,110 @@
+
+```
+$ wp plugin install rest-api --activate
+Installing WordPress REST API (Version 2) (2.0-beta13)
+Downloading install package from https://downloads.wordpress.org/plugin/rest-api.2.0-beta13.zip...
+Unpacking the package...
+Installing the plugin...
+Plugin installed successfully.
+Activating 'rest-api'...
+Success: Plugin 'rest-api' activated.
+```
+
+`wp transient` ([doc](http://wp-cli.org/commands/transient/)) lets you delete one or all transients:
+
+```
+$ wp transient delete-all
+Success: 34 transients deleted from the database.
+```
+
+For a complete introduction, read the [Quick Start guide](http://wp-cli.org/docs/quick-start/). If you already feel comfortable with the basics, jump into the [complete list of commands](http://wp-cli.org/commands/) for managing themes and plugins, importing and exporting data, performing database search-replace operations and more.
+
+## Installing
+
+The recommended way to install WP-CLI is to download the Phar build, mark it executable and place it in your PATH.
+
+Before you install though, please make sure your environment meets the minimum requirements:
+
+- UNIX-like environment (OS X, Linux, FreeBSD, Cygwin); limited support in Windows environment
+- PHP 5.3.29 or later
+- WordPress 3.7 or later
+
+Download the [wp-cli.phar](https://raw.github.com/wp-cli/builds/gh-pages/phar/wp-cli.phar) using `wget` or `curl`. For example:
+
+```
+$ curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+```
+
+Check if it is working:
+
+```
+$ php wp-cli.phar --info
+```
+
+To use it from the command line by typing `wp`, make the file executable and move it to somewhere in your PATH. For example:
 
-Installing
-------------
-If you just want to use WP-CLI, see <http://wp-cli.org/#install>.
+```
+$ chmod +x wp-cli.phar
+$ sudo mv wp-cli.phar /usr/local/bin/wp
+```
 
-If you want to hack on WP-CLI, see [CONTRIBUTING.md](CONTRIBUTING.md).
+If WP-CLI installed successfully, you should see something like this when you run `wp --info`:
 
-Where can I get more info?
---------------------------
-For documentation and examples, check out [wp-cli.org](http://wp-cli.org/) and the [documentation portal](http://wp-cli.org/docs/).
+```
+$ wp --info
+PHP binary:    /usr/bin/php5
+PHP version:    5.5.9-1ubuntu4.14
+php.ini used:   /etc/php5/cli/php.ini
+WP-CLI root dir:        /home/wp-cli/.wp-cli
+WP-CLI packages dir:    /home/wp-cli/.wp-cli/packages/
+WP-CLI global config:   /home/wp-cli/.wp-cli/config.yml
+WP-CLI project config:
+WP-CLI version: 0.23.0
+```
 
-Also, WordPress Answers has a growing list of [WP-CLI related questions](http://wordpress.stackexchange.com/questions/tagged/wp-cli).
+Now that you've got WP-CLI, read the [Quick Start](http://wp-cli.org/docs/quick-start/) guide. Or view the [full installation guide](http://wp-cli.org/docs/installing/) with alternative installation methods.
```

Comment:
If we link to alternative installation methods in the beginning of this section, I don't think we need this paragraph.